ios/SalesforceReact/SFMobileSyncReactBridge.m

```diff
@@ -50,8 +50,8 @@ @implementation SFMobileSyncReactBridge
 
 RCT_EXPORT_METHOD(getSyncStatus:(NSDictionary *)args callback:(RCTResponseSenderBlock)callback)
 {
-    NSNumber *syncId = (NSNumber *) [args nonNullObjectForKey:kSyncIdArg];
-    NSString *syncName = (NSString *) [args nonNullObjectForKey:kSyncNameArg];
+    NSNumber *syncId = (NSNumber *) [args sfsdk_nonNullObjectForKey:kSyncIdArg];
```

Comment:
@wmathurin or @bbirman - I haven't run these tests before on my workstation, but I noticed I had to update some method names for the test suite to compile.  Is this expected and simply some code that hasn't been updated from upstream changes?  Let me know ⏳

Reply:
This [PR](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3666) got rid of deprecated methods for 12.0.

Reply:
Thanks, @wmathurin - It looks like I was able to catch all the downstream impacts of the pull request you mentioned here and get a passing run in CI ⭐️